### PR TITLE
update mno color scheme to use color for unspecified operator in MNO colors

### PR DESCRIPTION
**Pull request by DanielMwendwa**
…colors

### CODE REVIEW

File: public/js/controllers/traffic_graphs_controller.js [hunk 1]
```diff
@@ -210,6 +210,10 @@ export class TrafficGraphsController {
         let mno_color_scheme = [],
             operators_with_color_identity = Object.keys(MNOColors);
 
+        let difference = operators.filter(x => !operators_with_color_identity.includes(x))[0];
```
Comment (IsaackMwenda) [inline public/js/controllers/traffic_graphs_controller.js]: what do you mean by difference?

Comment (DanielMwendwa) [reply public/js/controllers/traffic_graphs_controller.js]: Operators in array 'operators' that are not in array 'operators_with_color_identity'

Comment (IsaackMwenda) [reply public/js/controllers/traffic_graphs_controller.js]: Thanks, operators_without_color_identity would be more appropriate.

**Review by IsaackMwenda — APPROVED** [review]

File: public/js/controllers/traffic_graphs_controller.js (hunk 1 above)
Comment (DanielMwendwa) [reply public/js/controllers/traffic_graphs_controller.js]: Updated thanks

**Pull request merged**